contributors/design-proposals/seccomp.md

```diff
@@ -214,6 +248,192 @@ profiles using the key
 `seccomp.security.alpha.kubernetes.io/allowedProfileNames`.  The value of this
 key should be a comma delimited list.
 
+### Spec
+
+We will start from the OCI specification. This API resource will be added to
+`settings.k8s.io` as an `alpha` resource.
+
+```
+// Seccomp represents syscall restrictions
```

Comment:
Kinda feels a lot like storage classes which are not namespaced. @pmorie what makes it feel like namespaced? Multi-tenancy?